Formula/sslscan.rb

```diff
@@ -17,10 +17,12 @@ class Sslscan < Formula
 
   def install
     system "make"
+    mkdir_p [bin, man1]
     system "make", "install", "PREFIX=#{prefix}"
   end
 
   test do
+    assert_match version.to_s, shell_output("#{bin}/sslscan --version")
```

Comment:
@mikemcquaid I must just still be traumatized by https://github.com/Homebrew/homebrew-science/pull/3538/files#diff-11c674f1c0660805e38b0c57290060f2R25